command/secrets/output/result.go

```diff
@@ -0,0 +1,62 @@
+package output
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+)
+
+type SecretsOutputResult struct {
+	Address   string `json:"address,omitempty"`
+	BLSPubkey string `json:"bls,omitempty"`
+	NodeID    string `json:"node_id,omitempty"`
+
+	outputNodeID    bool `json:"-"`
+	outputBLS       bool `json:"-"`
+	outputValidator bool `json:"-"`
+}
+
+func (r *SecretsOutputResult) GetOutput() string {
+	var buffer bytes.Buffer
+
+	if r.outputNodeID {
+		buffer.WriteString(r.NodeID)
+
+		return buffer.String()
+	}
+
+	if r.outputValidator {
+		buffer.WriteString(r.Address)
+
+		return buffer.String()
+	}
+
+	if r.outputBLS {
+		buffer.WriteString(r.BLSPubkey)
+
+		return buffer.String()
+	}
+
+	vals := make([]string, 0, 3)
+
+	vals = append(
```

Comment:
Nitpick: 
Because we don't have a guard here to check whether address and nodeId exists, if data-dir does not exists the output would look like this
```
[SECRETS OUTPUT]
Public key (address) = 0x0000000000000000000000000000000000000000
Node ID              = <none>
```

For NodeID it's simple we can add a same guard as you added for blsPubKey

```
if r.NodeID != "" {
		vals = append(vals, fmt.Sprintf("Node ID|%s", r.NodeID))
}
```

For address, it's a bit trickier because once you use types.Address.String(), if the address does not exist it will convert it into 0x0 address (types.ZeroAddress). We can add a guard 

```
if r.Address != types.ZeroAddress.String() {
		vals = append(
			vals,
			fmt.Sprintf("Public key (address)|%s", r.Address),
		)
	}
```
because we are sure that no node address can be ZeroAddress. 

Another option is to print an error if the dir does not exist but i think it's more complicated and doesn't add much to ux

Reply:
Discussed the behavior to only print out the line if the respective secrets exist. May help with identifying missing keys in the future implicitly. 

```sh
$ ./polygon-edge secrets output --data-dir /not/here

[SECRETS OUTPUT]

```